packages/host/app/services/realm.ts

```diff
@@ -180,9 +228,9 @@ class RealmResource {
       let isPublic = Boolean(
         response.headers.get('x-boxel-realm-public-readable'),
       );
-      this.meta = { info, isPublic };
+      this.info = new TrackedObject({ ...info, isIndexing: false, isPublic });
```

Comment:
`isPublic` and `info` might be updated independently, but we only ever retrieve them once right here at the same time.